File: DATN-Fashion-Shop-BE/src/main/java/com/example/DATN_Fashion_Shop_BE/dto/response/store/staticsic/StoreDailyRevenueResponse.java
package com.example.DATN_Fashion_Shop_BE.dto.response.store.staticsic;

import lombok.*;

@Data
@Builder
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class StoreDailyRevenueResponse {
    private Integer day;
    private Integer month;
    private Integer year;
    private Double totalRevenue;
}

File: FE/DATN-Fashion-Shop-FE/src/app/dto/store/StoreDailyRevenueResponse.ts
import {UserResponse} from '../Response/user/user.response';
import {StoreOrderDetailResponse} from './StoreOrderDetailResponse';
import {StoreOrderStatusResponse} from './StoreOrderStatusResponse';

export interface StoreDailyRevenueResponse {
  day: number;
  month: number;
  year: number;
  totalRevenue: number;
}
